src/components/VAudioTrack/VAudioTrack.vue

```diff
@@ -129,6 +134,10 @@ export default defineComponent({
     const status = ref<AudioStatus>('paused')
     const currentTime = ref(0)
 
+    // We need to keep track if there was any error while playing the audio track.
+    let hasError = false // We initialize it to false, because we assume that this is a new track.
+    let messageError = '' // The error message is currently empty.
```

Comment:
Heads-up: these should be `ref` objects if they're going to cause re-renders.

```suggestion
    // We need to keep track if there was any error while playing the audio track.
    const hasError = ref(false) // We initialize it to false, because we assume that this is a new track.
    const messageError = ref('') // The error message is currently empty.
```

Reply:
Note that their usages will also have to be updated (to set `.value` for example)